src/index.js

```diff
@@ -36,6 +36,21 @@ class Stack {
     return this.elements[this.elements.length - 1];
   }
 
+  copyTo(array, index) {
```

Comment:
Your implementation dosen't work. Read documentation and fix method.
https://docs.microsoft.com/ru-ru/dotnet/api/system.collections.stack.copyto?view=netframework-4.7.2